extensions/WSLExtension/DistributionDefinitions/DistributionDefinition.yaml

```diff
@@ -0,0 +1,74 @@
+# This yaml file is used to map a distributions name to it logo file in the WslAssets folder. These distribution names

```

Comment:
If there are plans to supply this list from outside of the package, consider wrapping this list with a parent element. That way we can add some common properties like this file version.

Reply:
Currently no plans as of now so I'll leave it as is, but I do like that idea though.